src/gate/printer/graphml.h

```diff
@@ -0,0 +1,19 @@
+#include "gate/model/gnet.h"
+
+#include <ostream>
+#include <set>
+#include <string>
+#include <vector>
```

Comment:
Reduce the library includes. If function declarations in the header file don't use the included library, the include should not appear here.